Makefile

```diff
@@ -124,6 +128,12 @@ cnf-tests-local:
 
 check-tests-nodesc:
 	@echo "Checking undocumented cnf tests"
+	# pinning to specific commits
```

Comment:
why we do this?
and at the top we take master?

this will make the `check-tests-nodesc` to always pass

Reply:
I also commented about that in the commit message

Reply:
so should we just tag a specific version also for the test??
instead of
```
export METALLB_OPERATOR_TARGET_COMMIT?=main
export SRIOV_NETWORK_OPERATOR_TARGET_COMMIT?=master
export CLUSTER_NODE_TUNING_OPERATOR_TARGET_COMMIT?=master
```

Reply:
@fedepaol want your idea here.

now that cnf-test is not a product anymore should we continue to maintain the list of tests with a description to anyone of them?

Reply:
We agreed that the suite points at the tip of the corresponding version, so it doesn't make sense imo to have documentation. We can't keep it up to date in an automated fashion, and the users will all be internals.
This seems a hack to make the CI happy, which I really don't like. I am in favour of ditching the docs completely.

Reply:
removed.